.npmrc

```diff
@@ -1 +1,2 @@
 @origyn-sa:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=YOUR_TOKEN
```

Comment:
@3oltan is this going to break github actions? I mean worse than it's already broken?

Reply:
reversed